kayenta-signalfx/src/test/java/com/netflix/kayenta/signalfx/metrics/SimpleSignalFlowProgramBuilderTest.java

```diff
@@ -58,13 +60,88 @@ public void test_that_the_program_builder_builds_the_expected_program() {
     assertEquals(expected, builder.build());
   }
 
+  @Test
+  public void test_that_the_program_builder_builds_the_expected_program_when_location_is_provided_via_default() {
```

Comment:
This test and the one just below seem to match except for where the location key is specified. Could combine them and use a `where:` block and `@Unroll` instead. Would also make it easier for a reader to tease out the differences.